app/views/projects/_change_path.html.erb

```diff
@@ -0,0 +1,57 @@
+<%= viral_card do |card| %>
+  <% card.with_header(title: t(:"projects.edit.advanced.path.title")) %>
+  <% card.with_section do %>
+    <%= form_with(model: @project, url: namespace_project_path, method: :patch) do |form| %>
+      <%= form.fields_for :namespace, include_id: false do |builder| %>
+        <div class="grid gap-4">
+          <p class="text-gray-600 dark:text-gray-400">
+            <%= t(:"projects.edit.advanced.path.description") %>
+          </p>
+          <% invalid_path = @project.errors.include?(:path) %>
```

Comment:
Currently no error displays if you try to change the path of a project to an existing path. For example make a new project  under `yersinia/yersinia-pseudotuberculosis` named `Outbreak 2022a`, then go to this project's settings and try to update it's url to `Outbreak 2022`. You should see no error displayed under the change project url section. 

The reason this is happening is because the update service for the project is updating the project namespace attributes and not the actual project. This should be `@project.namespace.errors.include?(:path)` and the error should display

Reply:
Thank you for catching this. Please see 3c0b5eeef648d6dda3a1ddcebdc4dce0f950284d.

Reply:
Thanks!